types/src/front/lib/assistant.ts

```diff
@@ -378,14 +381,28 @@ export const CLAUDE_3_5_SONNET_DEFAULT_MODEL_CONFIG: ModelConfigurationType = {
   supportsVision: true,
   toolUseMetaPrompt: ANTHROPIC_TOOL_USE_META_PROMPT,
 };
+export const CLAUDE_3_5_HAIKU_DEFAULT_MODEL_CONFIG: ModelConfigurationType = {
+  providerId: "anthropic",
+  modelId: CLAUDE_3_5_HAIKU_20241022_MODEL_ID,
+  displayName: "Claude 3.5 Haiku",
+  contextSize: 180_000,
+  recommendedTopK: 32,
+  recommendedExhaustiveTopK: 128, // 65_536
+  largeModel: false,
```

Comment:
note: it's 4x the price of the old haiku (1/3 the price of sonnet) so not that cheap

Reply:
Yeah but we don't really rely on that that much. Only for legacy TEST_FREE_PLAN users which we plan to remove soon anyway :+1: